docs/cloud/pouta/vm-lifecycle.md

```diff
@@ -45,6 +45,10 @@ may benefit from the paused state but modern workflows generally do
 not use this state. Pausing a virtual machine is billed in the same
 way as an **active** state virtual machine.
 
+!!! warning
+
+    A *paused* virtual machine will be powered off during maintenance.
```

Comment:
Could it be something slightly more soft like
```
Please note that paused virtual machines cannot be live-migrated. In case of maintenance on the hypervisor hosting them, they will be powered off.
```

Reply:
I like Matteo's suggestion.